src/lro/lroPolicy.ts

```diff
@@ -24,11 +25,9 @@ class LROPolicy extends BaseRequestPolicy {
     webResource: WebResource
   ): Promise<HttpOperationResponse> {
     let result = await this._nextPolicy.sendRequest(webResource);
+    const _lroData = getLROData(result);
 
-    if (webResource.shouldDeserialize !== undefined) {
-      const _lroData = getLROData(result);
-      result.parsedBody = { ...result.parsedBody, _lroData };
-    }
+    Object.defineProperty(result, LROSYM, { value: _lroData });
```

Comment:
I don't think using `defineProperty` is necessary here, since symbols aren't enumerable in the same way normal properties are.

This should be equivalent:
```suggestion
    result[LROSYM] = _lroData;
```